src/LoginRouter.js

```diff
@@ -348,11 +350,19 @@ function (BaseLoginRouter,
     },
 
     enrollTotpFactor: function (provider, factorType) {
-      this.render(EnrollTotpController, {
-        provider: provider.toUpperCase(),
-        factorType: factorType,
-        Beacon: FactorBeacon
-      });
+      if (factorType === 'token:hotp') {
```

Comment:
I am using a generic route here. Even if I add one it would be the same pattern as the generic route for totp. I guess ordering might help but I guess it is safe to keep it here.

Reply:
a lot of our routes have the same pattern of `:provider/:factorType`, just take a look at all that start with `signin/enroll/*`.
For HOTP, we know this is a different factor than TOTP (different type, different controller). Also, this function is named `enrollTotpFactor`, so it shouldn't handle enrolling totp.
It seems cleaner and better organized to me if we add the route:
```
'signin/enroll/custom/token:hotp': 'enrollHotpFactor',
'signin/enroll/:provider/:factorType': 'enrollTotpFactor',
```